server/routes/calendar.js

```diff
@@ -1,6 +1,42 @@
 var express = require("express");
 var router = express.Router();
 
+var savedAvails = [];
+router.post("/save", (req, res) => {
```

Comment:
Is this endpoint actually doing anything? Can we delete? It looks like the endpoint below this is what is actually doing the saving to the database.